test/bdd/features/issuer_e2e_api.feature

```diff
@@ -10,11 +10,16 @@ Feature: Issuer VC REST API
 
   @issueCred_api
   Scenario: Issue Credential API
-    Given Public key stored in "publicKey" variable generated by calling Issuer Service Generate Keypair API
-    And   A new DID Document is created using the public key stored in "publicKey" and store the generate DID in "did" variable
-    Then  Verify the proof value generated using the Issuer Service Issue Credential API with the DID stored in "did" variable
+    Given "University" has a DID
+    Then  "University" application service verifies the credential created by Issuer Service issueCredential API with it's DID
 
   @composeAndIssueCred_api
   Scenario: Compose and Issue Credential API
-    Given "University" has a did
-    Then  "University" application service verifies the credential created by Issuer Service Issue Credential API
+    Given "University" has a DID
+    Then  "University" application service verifies the credential created by Issuer Service composeAndIssueCredential API with it's DID
```

Comment:
we should not be using composeAndIssueCredential in BDD test - it should be human readable. 

Given "University" has a DID
Then  "University" application service verifies that the credential created by Issuer Service is created with it's DID

Reply:
I'll update in next PR.